sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -47,8 +50,9 @@ import {
 } from "./internal/helpers";
 import { tracingPolicy } from "@azure/core-http";
 import { Spanner } from "./internal/tracingHelpers";
-import { GetKeyValuesResponse } from "./generated/src/models";
+import { GetKeyValuesResponse, AppConfigurationOptions } from "./generated/src/models";
 import { syncTokenPolicy, SyncTokens, SyncTokenHeaderName } from './internal/synctokenpolicy';
+import { DeserializationContentTypes } from '@azure/core-http/es/lib/policies/deserializationPolicy';
```

Comment:
This makes me nervous because you're importing something from another package based on that package's folder structure. core-http should probably export this from it's entry point since it does export the deserializationPolicy function.

Do you need to import this to get type-safety?

Reply:
I am equally nervous. Let me see why that's happening.

Reply:
Removed it - apparently we were always just using the right "shape" so I just used the constant, rather than passing it in.